### PR TITLE
Migrated from moment to luxon(#25k8h53)

### PR DESCRIPTION
**Pull request by disha1202**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -17,4 +17,12 @@ const showToast = async (message: string) => {
       return toast.present();
 }
 
-export { showToast, hasError }
\ No newline at end of file
+const handleDateTimeInput = (dateTimeValue: any) => {
```
Comment (adityasharma7) [inline src/utils/index.ts:20]: I think we should use this method thoughout while taking date time inputs

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/utils/index.ts [hunk 2]
```diff
@@ -17,4 +18,17 @@ const showToast = async (message: string) => {
...
 }
 
-export { showToast, hasError }
\ No newline at end of file
+const handleDateTimeInput = (dateTimeValue: any, inputType: string) => {
+  // TODO Handle it in a better way
+  // Remove timezone and then convert to timestamp
+  // Current date time picker picks browser timezone and there is no supprt to change it
+  let dateTime;
+  if (inputType === 'orderedBefore' || inputType === 'promisedBefore') {
```
Comment (adityasharma7) [inline src/utils/index.ts:26]: I think we should handle it differently, we should pass the input format and process date time accordingly

File: src/store/modules/user/actions.ts [hunk 3]
```diff
@@ -82,6 +81,9 @@ const actions: ActionTree<UserState, RootState> = {
    */
   async getProfile ( { commit }) {
     const resp = await UserService.getProfile()
+    if (resp.data.userTimeZone) {
```
Comment (adityasharma7) [inline src/store/modules/user/actions.ts:84]: This should be after we check if status is 200

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/utils/index.ts (hunk 2 above)
Comment (disha1202) [reply src/utils/index.ts:26]: Made the requested change

**Comment by k2maan** [comment]
Implemented already.

**Pull request closed without merging**